packages/editor-ui/src/composables/useWorkflowHelpers.ts

```diff
@@ -115,7 +115,7 @@ export function resolveParameter(
 			false,
 			undefined,
 			'',
-		) as IDataObject;
+		) as T;
```

Comment:
could we get rid of this type coercion instead?

Reply:
Not easily, `expression.getParameterValue` returns `NodeParameterValueType`, but that type is to narrow for `resolveParameter`, `resolveParameter` is being used to resolve any value/expression/object of expressions, etc.